Docker/itksoftwareguide-bin/README.md

```diff
@@ -1 +1,21 @@
+Description
+-----------
+
 Holds the volume with the build for the Software Guide.
+
+Volumes
+-------
+
+Source folder: /home/itk/src/ITKSoftwareGuide
+Binary folder: /home/itk/bin/ITKSoftwareGuide
+
+Usage
+-----
+
+Build in the docker container:
+  ./run.sh itksoftwareguide-bin
+
+Use ITKSoftwareGuide outside of the docker container:
+  ./run.sh itksoftwareguide-bin \
+    -v ${HostITKSoftwareGuidSourceDirectory}:/home/itk/src/ITKSoftwareGuide \
+    -v ${HostITKSoftwareGuidBinaryDirectory}:/home/itk/bin/ITKSoftwareGuide
```

Comment:
> that seems odd as now as all versions on the download page do support it

Since there was still confusion regarding https/ssl support (e.g this morning), I was thinking that adding a simple sentence to the download page would help others.